tests/utils/simulate.js

```diff
@@ -143,6 +143,46 @@
         element.dispatchEvent(event);
     }
 
+    function sendTouchEvent2(x1, y1, x2, y2, element, eventType) {
```

Comment:
I'd like to see this renamed to  sendTouchEvents and it should accept an array of coordinates